docs/source/conf.py

```diff
@@ -64,7 +64,8 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+# Mike Bopf - 01Dec2022 - setting default to English to avoid warning.
```

Comment:
Please change to "Setting default to English to avoid warning" without the prefix information. Who made the change and when is recorded via the git commits. 

On the command-line if you run `git blame file_name` you will see who was the last person to touch each line of the file. On github you can navigate to the source file and on the upper right corner there is a "blame" button which will display this info.